kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/wifi/HostapdManager.java

```diff
@@ -156,4 +154,58 @@ public static String getHostapdConfigFileName(String ifaceName) {
 		}
 		return sb.toString();
 	}
+	
+	private static String privGetHostapdConfigFileName(String ifaceName) {
```

Comment:
@ibinshtok do you think we could merge the privGetHostapdConfigFileName with the already existing getHostapdConfigFileName? Maybe checking if a certain path exists and returning the correct path back. Is it feasible?